### PR TITLE
change bool to str value

### PR DESCRIPTION
**Pull request by bissenbay**
Fix the valye of `KAFKA_TOPIC_CONSUMER_ENABLE_AUTO_COMMIT`

### CODE REVIEW

File: cve-update/overlays/test/cronjob.yaml [hunk 1]
```diff
@@ -87,7 +87,7 @@ spec:
                 - name: KAFKA_CAFILE
                   value: "/mnt/secrets/kafka_ca.crt"
                 - name: KAFKA_TOPIC_CONSUMER_ENABLE_AUTO_COMMIT
-                  value: false
+                  value: "False"
```
Comment (harshad16) [inline cve-update/overlays/test/cronjob.yaml:90]:
```suggestion
                  value: "false"
```

Comment (bissenbay) [reply cve-update/overlays/test/cronjob.yaml]: thanks, fixed

**Review by harshad16 — APPROVED** [review]
lgtm

**Pull request merged**